widgets/search_bar.py

```diff
@@ -102,25 +102,28 @@ def clear_all(self) -> None:
 
     def _setup_layout(self) -> None:
         self.setLayout(layout := QVBoxLayout())
+        layout.addWidget(self.bar)
         layout.addWidget(self.opts)
         layout.addWidget(self.remote_opts)
-        layout.addWidget(self.bar)
```

Comment:
Why is the search bar shifted up?

Reply:
I've decided to commit it straight to main. 

It's just that those small changes are not related to the PR itself, they just make it harder to read the diff.